ant-evm/src/amount.rs

```diff
@@ -67,7 +67,7 @@ impl AttoTokens {
 
     /// Converts the Nanos into bytes
     pub fn to_bytes(&self) -> Vec<u8> {
```

Comment:
Here the comment still refers to Nanos

Reply:
Yea I missed that one. This file was failing tests mind you, that's the only reason I ended up making changes to it.